From 5f497cba41785fc5532633811001958f99573f9b Mon Sep 17 00:00:00 2001
From: Cole Robinson <crobinso@redhat.com>
Date: Tue, 09 Jul 2013 21:50:05 +0000
Subject: console: Don't double create spice display

I can't reproduce, but a couple gentoo users have issues that this
fixes:

https://bugzilla.redhat.com/show_bug.cgi?id=981975
---
diff --git a/virtManager/console.py b/virtManager/console.py
index 51175c0..553c022 100644
--- a/virtManager/console.py
+++ b/virtManager/console.py
@@ -536,7 +536,8 @@ class SpiceViewer(Viewer):
             channel.connect_after("channel-event", self._main_channel_event_cb)
             return
 
-        if type(channel) == SpiceClientGLib.DisplayChannel:
+        if (type(channel) == SpiceClientGLib.DisplayChannel and
+            not self.display):
             channel_id = channel.get_property("channel-id")
 
             if channel_id != 0:
--
cgit v0.9.1
